examples/account-signature-checks/src/lib.rs

```diff
@@ -0,0 +1,87 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+//! A basic example showing how to retrieve account keys, and check account
+//! signatures.
+use concordium_std::*;
+use core::fmt::Debug;
+
+#[derive(Debug, PartialEq, Eq, Reject, Serial, SchemaType)]
+enum Error {
+    /// Failed parsing the parameter.
+    #[from(ParseError)]
+    ParseParamsError,
+    /// Account that we wanted was not present.
+    MissingAccount,
+    /// Signature data was malformed.
+    MalformedData,
+}
+
+impl From<CheckAccountSignatureError> for Error {
+    fn from(value: CheckAccountSignatureError) -> Self {
+        match value {
+            CheckAccountSignatureError::MissingAccount => Self::MissingAccount,
+            CheckAccountSignatureError::MalformedData => Self::MalformedData,
+        }
+    }
+}
+
+impl From<QueryAccountPublicKeysError> for Error {
+    fn from(QueryAccountPublicKeysError: QueryAccountPublicKeysError) -> Self {
+        Self::MissingAccount
+    }
+}
+
+/// We don't need state for this specific demonstration.
+#[derive(Serialize)]
+struct State {}
+
+/// Init function that creates a new smart contract.
+#[init(contract = "account_signature_checks")]
+fn init<S: HasStateApi>(
+    _ctx: &impl HasInitContext,
+    _state_builder: &mut StateBuilder<S>,
+) -> InitResult<State> {
+    Ok(State {})
+}
+
+#[derive(Deserial, SchemaType)]
+struct CheckParam {
+    address: AccountAddress,
+    sigs:    AccountSignatures,
+    #[concordium(size_length = 4)]
```

Comment:
This is the defaut size_length for Vec, so this attribute does not change anything

Reply:
It does not have to change anything to be useful. To me this is reassuring since it makes it explicit what is expected and I don't have to go find it somewhere else.